openedx/core/djangoapps/user_api/views.py

```diff
@@ -889,6 +922,96 @@ def _apply_third_party_auth_overrides(self, request, form_desc):
                         restrictions={}
                     )
 
+    def _add_organization_field(self, form_desc, required=True):
+        """
+        Add a Organization field to a form description.
+
```

Comment:
cant add these fields from http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/customize_registration_page.html